build.gradle

```diff
@@ -1,21 +1,27 @@
-plugins {
-    id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.10'
-    id 'org.jetbrains.kotlin.plugin.allopen' version '1.3.10'
-    id 'org.jetbrains.kotlin.plugin.spring' version '1.3.10'
-    id 'org.springframework.boot' version '2.0.4.RELEASE'
-    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}"
+        classpath "org.jetbrains.kotlin:kotlin-allopen:${kotlinVersion}"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
+        classpath "io.spring.gradle:dependency-management-plugin:${springDependencyManagementVersion}"
+    }
 }
 
-group 'sk.vrto'
```

Comment:
One random comment. While the `group` will be inherited it _might_ be worth setting submodule project names. 

Otherwise you final bundle (unless you wrap as fat-jar) will end up having `user.jar`, `customer.jar`. Having prefix such as `my-project-user.jar` could help.

Again, might not apply.

Reply:
Thx - I was shooting for a fat JAR, but if I agree that having a module named `customers.jar` without any prefix isn't ideal!